publish_request_test.go

```diff
@@ -414,19 +405,19 @@ func TestPublishStore(t *testing.T) {
 func TestPublishEncrypt(t *testing.T) {
 	assert := assert.New(t)
 
-	pnconfig.CipherKey = "testCipher"
+	pn := NewPubNub(NewDemoConfig())
+	pn.Config.UseRandomInitializationVector = false
+	pn.Config.CipherKey = "testCipher"
 
-	opts := &publishOpts{
-		Channel: "ch",
-		Message: "hey",
-		pubnub:  pubnub,
-	}
+	opts := newPublishOpts(pn, pn.ctx)
+	opts.Channel = "ch"
+	opts.Message = "hey"
 
 	path, err := opts.buildPath()
 	assert.Nil(err)
 
 	assert.Equal(
-		"/publish/pub_key/sub_key/0/ch/0/%22%2Bc52pEK3TCTpuEjEFzukRw%3D%3D%22", path)
+		"/publish/demo/demo/0/ch/0/%22MnwzPGdVgz2osQCIQJviGg%3D%3D%22", path)
```

Comment:
Is this an intentional change? I don't get where the implementation changes ;/.

Reply:
Yes, this was an intentional change. The default `Serialize: true` was added by `newPublishOpts` constructing method but it was not used in the test previously, therefore the output should change